test-harness/modules/mocker/common.py

```diff
@@ -53,6 +53,13 @@ def _create_common_endpoints():
         res.append(ReflectingTcpIpEndpoint(ip='127.0.0.1', port=8181))
         # mesos
         res.append(MesosEndpoint(ip='127.0.0.2', port=5050))
+        # additional mesos instances for leader DNS changes tests
+        res.append(MesosEndpoint(ip='127.0.0.3', port=5050))
+        # mesos non-leading master that runs Mesos and AR
+        res.append(ReflectingTcpIpEndpoint(ip='127.0.0.10', port=5050))
+        res.append(ReflectingTcpIpEndpoint(ip='127.0.0.10', port=80))
```

Comment:
I understand but Mesos mocked servers running on port `5050` on both IP addresses aren't `ReflectingTcpIpEndpoint` but rather `MesosEndpoint`. This means that we couldn't use parametrized test and single `generic_correct_upstream_dest_test` for both type of URLs.